modules/nf-core/trinity/main.nf

```diff
@@ -35,15 +35,15 @@ process TRINITY {
     if (!task.memory) {
         log.info '[Trinity] Available memory not known - defaulting to 7GB. Specify process memory requirements to change this.'
     } else {
-        avail_mem = task.memory.giga
+        avail_mem = (task.memory.mega*0.8).intValue()
```

Comment:
Can't comment on that line but need to change the `def avail_mem = 7` to megabytes